### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Classylilbadassy69** [community/community#65915] [Actions]
### Select Topic Area

General

### Body

I'm sorry I don't know where to put this have a hacker problem and there 3 steps ahead of me. I have not done anything on my account except for try to secure it the tokens are not me they are using them to lock me out of all my accounts. I would like two of the pictures on here behind mine are a cat I don't know yes she's been going on for six months maybe longer I can't get them to stop hacking everything. And I don't understand any of this stuff making it worse.....

**Comment by Classylilbadassy69** [comment]
https://github.com/microsoft/vscode-js-debub

https://app.vssps.visualstudio.com/_signedin

@ghost@Classylilbadassy69	
GitHub System – oauth_access.regenerate
Regenerated a token for Classylilbadassy69 ending in ww34kdIM for the Microsoft-Corporation OAuth app.
3 hours ago …

https://login.live.com/oauth20_authorize.srf?lc=1033&estsfed=1&client_id=51483342-085c-4d86-bf88-cf50c7252078&response_mode=form_post&cobrandid=31ffa2bb-717b-40bc-831f-f7f1a66612db&pid=15216&fci=499b84ac-1321-427f-aa17-267ca6975798&wsucxt=1&code=77c994e7283fbac07e8a&response_type=code&state=B8A901F053C0D721&scope=openid+profile+email+offline_access&redirect_uri=https%3a%2f%2flogin.microsoftonline.com%2fcommon%2ffederation%2foauth2msa&mkt=EN-US&uaid=4fe443fef79f4102a49e99df5dba2f6e&contextid=B8A901F053C0D721&opid=B8A901F053C0D721&opidt=1693825370&nw=4G&route=C106_BAY


https://login.microsoftonline.com/common/federation/oauth2msa

I'm pretty sure these aren't supposed to look that way though I'd post them I don't know why I did I guess craft paper straws or maybe somebody might know how to really stop being from getting on here....  I don't think I'll be able to get back on here but ok either way thank you.

**Comment by nerdistry** [comment]
Change Passwords:

Change the passwords for all your accounts, starting with your email account. Use strong, unique passwords for each account. Consider using a password manager to generate and store complex passwords.
Enable Two-Factor Authentication (2FA):

Enable 2FA for all your accounts that support it. This adds an extra layer of security, making it much harder for hackers to gain access even if they have your password.
Check for Compromised Accounts:

Use services like "Have I Been Pwned" to check if your email address has been involved in any data breaches. If it has, change passwords on affected accounts immediately.
